### PR TITLE
tlmgr-pinning: add page

### PR DESCRIPTION
**Pull request by laurenszlosek**
- [ X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ X] The page(s) have at most 8 examples.
- [X ] The page description(s) have links to documentation or a homepage.
- [ X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/tlmgr-pinning.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`pinning show`
+
+- Pin the packages matching the packages(s) to the repository repository:
+
+`pinning add {{repository}} {{package(s)}}`
+
+- Any packages recorded in the pinning file matching the <package>s for the given repository are removed:
+
+`pinning remove {{repository}} {{package(s)}}`
```
Comment (kbdharun) [inline pages/common/tlmgr-pinning.md:16]:
```suggestion
`tlmgr pinning remove {{repository}} {{package1 package2 ...}}`
```

File: pages/common/tlmgr-pinning.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# tlmgr pinning
+
+> The pinning action manages the pinning file.
+> More information: <https://www.tug.org/texlive/doc/tlmgr.html#pinning>.
+
+- Show the current pinning data:
+
+`pinning show`
+
+- Pin the packages matching the packages(s) to the repository repository:
```
Comment (kbdharun) [inline pages/common/tlmgr-pinning.md:10]:
```suggestion
- Pin the matching the packages to the given repository:

File: pages/common/tlmgr-pinning.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`pinning add {{repository}} {{package(s)}}`
```
Comment (kbdharun) [inline pages/common/tlmgr-pinning.md:12]:
```suggestion
`tlmgr pinning add {{repository}} {{package1 package2 ...}}`
```

File: pages/common/tlmgr-pinning.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# tlmgr pinning
+
+> The pinning action manages the pinning file.
+> More information: <https://www.tug.org/texlive/doc/tlmgr.html#pinning>.
+
+- Show the current pinning data:
+
+`pinning show`
```
Comment (kbdharun) [inline pages/common/tlmgr-pinning.md:8]:
```suggestion
`tlmgr pinning show`
```

File: pages/common/tlmgr-pinning.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`pinning add {{repository}} {{package(s)}}`
+
+- Any packages recorded in the pinning file matching the <package>s for the given repository are removed:
+
+`pinning remove {{repository}} {{package(s)}}`
+
+- Remove all pinning data for repository repo:
+
+`pinning remove repo --all`
```
Comment (kbdharun) [inline pages/common/tlmgr-pinning.md:20]:
```suggestion
`tlmgr pinning remove {{repository}} --all`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM :)
Thank you @laurenszlosek

**Pull request merged**